app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,62 @@
+class CommentsController < ApplicationController
+  before_action :set_comment, only: [:edit, :update, :destroy]
+  before_action :check_permission, only: [:edit, :destroy]
+
+  def new
```

Comment:
This is not needed. The updates will be done through an ajax request (POST /comment)